indico/migrations/versions/20181213_1110_cbe630695800_add_room_principals_table.py

```diff
@@ -0,0 +1,204 @@
+            booking_group = _get_attr_value(conn, room_id, booking_group_attr_id)
+        if manager_group_attr_id is not None:
+            manager_group = _get_attr_value(conn, room_id, manager_group_attr_id)
+
+        if not booking_group:
+            conn.execute('UPDATE roombooking.rooms SET protection_mode = %s WHERE id = %s',
+                         (ProtectionMode.public.value, room_id))
+        _create_acl_entry(conn, room_id, user_id=owner_id, full_access=True)
+        if booking_group:
+            group_kwargs = _group_to_kwargs(booking_group)
+            if group_kwargs is None:
+                print 'WARNING: Invalid booking group: {}'.format(booking_group)
+            else:
+                permission = 'prebook' if reservations_need_confirmation else 'book'
+                _create_acl_entry(conn, room_id, permissions={permission}, **group_kwargs)
+        if manager_group:
+            group_kwargs = _group_to_kwargs(manager_group)
+            if group_kwargs is None:
+                print 'WARNING: Invalid manager group: {}'.format(manager_group)
+            else:
+                _create_acl_entry(conn, room_id, full_access=True, **group_kwargs)
+
+
+def _create_attribute(conn, name, title):
+    res = conn.execute('''
+        INSERT INTO roombooking.room_attributes (name, title, is_hidden)
+        VALUES (%s, %s, false)
+        RETURNING id
+    ''', (name, title))
+    return res.fetchone()[0]
+
+
+def _set_attribute_value(conn, room_id, attribute_id, value):
+    res = conn.execute('SELECT value FROM roombooking.room_attribute_values WHERE room_id = %s AND attribute_id = %s',
+                       (room_id, attribute_id))
+    if not res.rowcount:
+        conn.execute('''
+            INSERT INTO roombooking.room_attribute_values
+            (room_id, attribute_id, value) VALUES
+            (%s, %s, %s)
+        ''', (room_id, attribute_id, json.dumps(value)))
+    elif res.scalar() != value:
+        conn.execute('''
+            UPDATE roombooking.room_attribute_values
+            SET value = %s
+            WHERE room_id = %s AND attribute_id = %s
+        ''', (json.dumps(value), room_id, attribute_id))
+
+
+def _downgrade_permissions():
+    conn = op.get_bind()
+    booking_group_attr_id = conn.execute('SELECT id FROM roombooking.room_attributes WHERE name = %s',
+                                         ('allowed-booking-group',)).scalar()
+    manager_group_attr_id = conn.execute('SELECT id FROM roombooking.room_attributes WHERE name = %s',
+                                         ('manager-group',)).scalar()
+    if booking_group_attr_id is None:
+        booking_group_attr_id = _create_attribute(conn, 'allowed-booking-group', 'Allowed Booking Group')
+    if manager_group_attr_id is None:
+        manager_group_attr_id = _create_attribute(conn, 'manager-group', 'Manager Group')
+
+    query = 'SELECT id, owner_id, reservations_need_confirmation, protection_mode from roombooking.rooms'
```

Comment:
I know it is something trivial but to keep it consistent maybe change `from` to `FROM`?